app/src/main/java/net/squanchy/schedule/view/SpeakerView.java

```diff
@@ -25,6 +35,7 @@ public SpeakerView(Context context, AttributeSet attrs, int defStyleAttr) {
     public SpeakerView(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
         super(context, attrs, defStyleAttr, defStyleRes);
 
+        imageLoader = ImageLoaderInjector.obtain(context).imageLoader();
```

Comment:
This means every instance of `SpeakerView` will have a new instance of `ImageLoaderComponent`, `ImageLoader` and so on, are we sure that's what you meant to do?